packages/grafana-data/package.json

```diff
@@ -34,6 +35,7 @@
     "@rollup/plugin-json": "4.0.3",
     "@rollup/plugin-node-resolve": "7.1.1",
     "@types/braintree__sanitize-url": "4.0.0",
+    "@types/d3-interpolate": "^1.3.1",
```

Comment:
Why is this dependency added to both deps and devDeps?

Reply:
good catch